### PR TITLE
Update template/upload copy

### PR DESCRIPTION
**Pull request by JordanGuinn**
## Background
This PR generally updates the text on the data template and initial file upload pages to inform users to use the new template, given we've done away with reporting periods.

## GitHub Issue
- https://github.com/ctoec/data-collection/issues/1497

## Validation Plan
- [x] Ensure links in the copy on the file upload page work as intended
- [x] Ensure copy on the file upload page matches the ticket
- [x] Ensure copy on the data template page matches the ticket

## Automated Testing
- [x] All unit tests are passing.

### CODE REVIEW

File: client/src/containers/TemplateDownload/TemplateDownload.tsx [hunk 1]
```diff
@@ -3,19 +3,36 @@ import { getH1RefForTitle } from '../../utils/getH1RefForTitle';
 import { CSVExcelDownloadButton } from '../../components/CSVExcelDownloadButton';
 import { ReactComponent as Image } from '../../images/SpreadsheetIllustration.svg';
 import { BackButton } from '../../components/BackButton';
+import useSWR, { responseInterface } from 'swr';
```
Comment (bamader) [inline client/src/containers/TemplateDownload/TemplateDownload.tsx:6]: _Wait_ we kept SWR around???

File: src/template/constants.ts [hunk 2]
```diff
@@ -3,8 +3,8 @@ import moment from 'moment';
 // UPDATE THESE IF YOU CHANGE THIS FILE!!
 // TODO: add pre-merge hook that updates these values (probably put them in a diff file)
 // automatically when this file gets updated
-export const TEMPLATE_VERSION = 2;
-export const TEMPLATE_LAST_UPDATED = moment.utc('05-27-2021', ['MM-DD-YYYY']);
+export const TEMPLATE_VERSION = 3;
+export const TEMPLATE_LAST_UPDATED = moment.utc('10-21-2021', ['MM-DD-YYYY']);
```
Comment (bamader) [inline src/template/constants.ts:7]: Is it worth maintaining which version of the template we're on? Do we use it specifically for anything, or just the template updated date?

File: client/src/containers/TemplateDownload/TemplateDownload.tsx (hunk 1 above)
Comment (JordanGuinn) [reply client/src/containers/TemplateDownload/TemplateDownload.tsx:6]: heh, remember we removed it as part of the roster redesign work

**Comment by nickbristow** [comment]
## Validation
- [ ] Ensure links in the copy on the file upload page work as intended
- [ ] Ensure copy on the file upload page matches the ticket
- [ ] Ensure copy on the data template page matches the ticket

**Pull request merged**